worker/uniter/runner/jujuc/relation-list.go

```diff
@@ -49,8 +50,9 @@ func (c *RelationListCommand) Info() *cmd.Info {
 
 func (c *RelationListCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.out.AddFlags(f, "smart", cmd.DefaultFormatters.Formatters())
-	f.Var(c.relationIdProxy, "r", "specify a relation by id")
+	f.Var(c.relationIdProxy, "r", "Specify a relation by id")
 	f.Var(c.relationIdProxy, "relation", "")
+	f.BoolVar(&c.ListRemoteApplication, "app", false, "List remote application instead of participating units")
```

Comment:
s/List remote application/List requiring application/

or participating application.

Reply:
The bug description seems to use the same terminology (remote application) so we should probably ask @petevg to clarify whether external users and/or stakeholders use different terminology and make sure that docs and hook tool usage are consistent.